.changeset/mighty-hairs-train.md

```diff
@@ -0,0 +1,61 @@
+---
+"@comet/admin": minor
+---
+
+Add a more actions menu helper component
+
+This is done to provide a simple api to create a more actions menu for a list of items. 
+
+**Example:**
+
+```tsx
+<MoreActionsMenu
```

Comment:
The more actions menu system seems quite similar to the existing [RowActions system](https://storybook.comet-dxp.com/?path=/story/docs-components-rowactions--page). 

Have you considered combining these systems into a single group of components?
Either by extending the RowActions system or by using the more actions menu system to replace the RowActions.

I think `RowActions` could relatively easily be extended to cover these use cases as well.
E.g. by adding a `moreMenuText` for the text next to the ellipsis icon and something like `<RowActionsGroupTitle />` for the titles above a set of items.

```tsx
<RowActionsMenu>
    <RowActionsMenu moreMenuText="More actions" selectionSize={2}>
        <RowActionsItem icon={<Move />}>Move</RowActionsItem>
        <Divider />
        <RowActionsGroupTitle>Some Group Title</RowActionsGroupTitle>
        <RowActionsItem icon={<Upload />}>Upload</RowActionsItem>
        <RowActionsItem icon={<Download />}>Download</RowActionsItem>
    </RowActionsMenu>
</RowActionsMenu>
```

<img width="229" alt="MoreActions Menu Implementation" src="https://github.com/vivid-planet/comet/assets/6264317/d6ef5bab-78d7-4656-8e03-bd143e8dcfba">

Reply:
> I think RowActions could relatively easily be extended to cover these use cases as well.

@johnnyomair what do you think?

Reply:
I think that's a good idea! RowActions might not be the best name then though

Reply:
`ActionsMenu` 🤷🏼‍♂️

Reply:
@thomasdax98 @nsams what do you think about @jamesricky's suggestion?

Reply:
sounds good, but I don't know enough details

What does this RowActions more than a https://mui.com/material-ui/react-menu/#basic-menu?